### PR TITLE
[i18n] add edit patient for all languages

### PR DESCRIPTION
**Pull request by marcosvega91**
I added the translation of edit patient for all languages

The label edit patient was not translated neither in english.

Regards
Marco

### CODE REVIEW

**Comment by jackcmeyer** [comment]
@marcorosas1991 Thanks for your contribution. It looks like an account that you used to make your first commit has not signed the CLA, yet. 

There are a couple ways to resolve the issue: 
* Add the email associated with the first commit to your GitHub account
* Amend the first commit to make the email the one that the CLA has bee signed with.

once this is finished, we can merge the PR.

**Comment by marcosvega91** [comment]
@jackcmeyer Hi, as default account for Git i used another email. I already amend the commit with the correct email and sign the CLA with GitHub account

File: src/locales/fr/translations/patients/index.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export default {
     patientsList: 'Liste des patients',
     viewPatients: 'Voir les patients',
     viewPatient: 'Voir le patient',
+    editPatient: 'Changer de patient',
```
Comment (tehKapa) [inline src/locales/fr/translations/patients/index.ts:7]: "Modifier le patient" is the correct translation.

File: src/locales/es/translations/patients/index.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ export default {
     label: 'Pacientes',
     viewPatients: 'Ver pacientes',
     viewPatient: 'Ver pacientes',
+    editPatient: 'Cambiar paciente',
```
Comment (tehKapa) [inline src/locales/es/translations/patients/index.ts:6]: "Editar paciente"  is the correct translation.

File: src/locales/ru/translations/patients/index.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ export default {
     label: 'Пациенты',
     viewPatients: 'Просмотр пациентов',
     viewPatient: 'Просмотр пациента',
+    editPatient: 'Сменить пациента',
```
Comment (tehKapa) [inline src/locales/ru/translations/patients/index.ts:6]: "Редактировать пациента" is the correct translation

**Comment by tehKapa** [comment]
@marcosvega91 we removed directory `in` and `zr` that didn't match ISO language codes. You should resolve the conflicts, thanks!

**Pull request merged**